ui/lib/core/addon/components/filter-input.hbs

```diff
@@ -3,9 +3,11 @@
   SPDX-License-Identifier: BUSL-1.1
 }}
 
-<div class="control {{unless @hideIcon 'has-icons-left'}}" data-test-filter-input-container>
-  <input class="filter input" ...attributes {{on "input" this.onInput}} {{did-insert this.focus}} data-test-filter-input />
-  {{#unless @hideIcon}}
-    <Icon @name="search" class="search-icon has-text-grey-light" data-test-filter-input-icon />
-  {{/unless}}
-</div>
\ No newline at end of file
+<Hds::Form::TextInput::Base
+  @type="search"
+  @id={{@id}}
```

Comment:
Since HDS components automatically generate an ID - I'm not sure we still need this?

Reply:
I don't know :) it may be that the ID is used somewhere, since this is a component with different instances

Reply:
I'm fairly confident we don't need it, but agree this can be addressed as follow-on cleanup!